lib/gps.js

```diff
@@ -432,6 +562,40 @@ GPS.prototype.parseNmeaSentence = function(sentence) {
 
 };
 
+/*
+ * SkyTraq Binary Message Protocol
+ * http://cdn.sparkfun.com/datasheets/Sensors/GPS/Venus/638/doc/AN0003_v1.4.19.pdf
+ */
+GPS.prototype.parseSkyTraqSentence = function(sentence) {
```

Comment:
If you have a broken out listen method you can just put the custom parser method on the CHIP object but I don't think that's a good idea...

I'm sure there will be other chips that use Skytraq (and multiple chips that use SiRF, Rockwell and Sony formats) but DRY is good so I have a proposition:

Let's create a PARSER object that wraps each of the parsing methods, and then the chip can specify which method on PARSER to use. i.e.:

````
PARSER = {
  NMEA: function(sentence) {},
  SKYTRAQ: function(sentence) { ... },
  etc...
}

// and then on CHIP

Venus638FLPx: {
  parse: PARSER.SKYTRAQ,
  baud: {
    value: 9600
  },
  ...
````